### PR TITLE
new folder to hold all icons

### LINKED ISSUE #58: ALL ICONS Component
| name  | about | title  | labels | assignees  |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| "👥 User story"  | Create a new user story  | ALL ICONS COMPONENT  | enhancement  | Gayle | 

## Story

- **As a** user

- **I Want** be able to see all the relevant icons rendered neatly on the page

- **So That** I can tell which icon represents what at a glance

## Acceptance Criteria

1. Given I am on the teams info page
   - When I see all the icons representing the teams different pages
   - Then I want to be able to tell what each icon represents for each team
  

## Implementation Notes

1.All the icons have been added but I am putting them all inside one div to make it easy and neater to use and render.
2.

## Checklist:

- [x] I have defined the user story
- [x] I have defined some acceptance criteria
- [x] I have included any implementation notes
- [ ] I have included all relevant links

### PR DESCRIPTION
**Pull request by Gayle-Thompson-Igwebike**
#58-All-Icons-Component-Gayle-Thompson-Igwebike

## Description

Created a new folder to hold all the icons for the teams page. Plus simple basic css. -  It probably wont show just now, because the other components need to be merged first.

## Related to #58 

Fixes #58 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/Icons-Components/All-IconsContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './All-IconsContainer.css';
+import SlackIconSvg from './SlackIconSvg';
```
Comment (AndrianaOS) [inline client/src/components/Icons-Components/All-IconsContainer.jsx]: Your paths dont point into any folders. They need to point to the file containing the images

**Review by AndrianaOS — DISMISSED** [review]
Please configure your paths. They crash the page when included in App.js as they dont have the right file path

File: client/src/components/Icons-Components/All-IconsContainer.jsx (hunk 1 above)
Comment (Gayle-Thompson-Igwebike) [reply client/src/components/Icons-Components/All-IconsContainer.jsx]: OK. Thank you.

**Comment by Gayle-Thompson-Igwebike** [comment]
The file paths have been resolved

**Review by AndrianaOS — APPROVED** [review]

File: client/src/components/Icons/DeployedAppIconSvg.jsx
Comment (Gayle-Thompson-Igwebike) [inline client/src/components/Icons/DeployedAppIconSvg.jsx:1]: Added correct links for each icon but I think they are causing conflicts so changed them back

**Pull request merged**